flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/assigners/ChunkSplitter.java

```diff
@@ -137,7 +133,14 @@ public ChunkSplitter(MySqlSchema mySqlSchema, MySqlSourceConfig sourceConfig, in
         }
 
         final List<ChunkRange> chunks;
-        if (isSplitColumnEvenlyDistributed(jdbc, tableId, splitColumn, min, max, chunkSize)) {
+        if (isSplitColumnEvenlyDistributed(
+                jdbc,
+                tableId,
+                splitColumn,
+                min,
+                max,
+                chunkSize,
+                sourceConfig.getEvenlyDistributionFactor())) {
```

Comment:
We don't need to pass as a parameter, because you can get the value in the method.

Reply:
no, here is a private method, but `isSplitColumnEvenlyDistributed` is static method